docs/sources/enterprise/saml.md

```diff
@@ -7,165 +7,126 @@ type = "docs"
+
+### Certificate and private key
 
 The SAML SSO standard uses asymmetric encryption to exchange information between the SP (Grafana) and the IdP. To perform such encryption, you need a public part and a private part. In this case, the X.509 certificate provides the public part, while the private key provides the private part.
 
-Grafana supports two ways of specifying both the `certificate` and `private_key`. Without a suffix (e.g. `certificate=`), the configuration assumes you've supplied the base64-encoded file contents. However, if specified with the `_path` suffix (e.g. `certificate_path=`) Grafana will treat it as a file path and attempt to read the file from the file system.
+Grafana supports two ways of specifying both the `certificate` and `private_key`. 
+* Without a suffix (`certificate` or `private_key`), the configuration assumes you've supplied the base64-encoded file contents. 
+* With the `_path` suffix (`certificate_path` or `private_key_path`), then Grafana treats the value entered as a file path and attempt to read the file from the file system.
+
+You can only use one form of each configuration option. Using multiple forms, such as both `certificate` and `certificate_path`, results in an error.
 
 ### IdP metadata
 
-Expanding on the above, we'll also need the public part from our IdP for message verification. The SAML IdP metadata XML tells us where and how we should exchange the user information.
+You also need to define the public part of the IdP for message verification. The SAML IdP metadata XML defines where and how Grafana exchanges user information.
 
-Currently, we support three ways of specifying the IdP metadata. Without a suffix `idp_metadata=` Grafana assumes base64-encoded XML file contents, with the `_path` suffix assumes a file path and attempts to read the file from the file system and with the `_url` suffix assumes an URL and attempts to load the metadata from the given location.
+Grafana supports three ways of specifying the IdP metadata. 
+* Without a suffix `idp_metadata`, Grafana assumes base64-encoded XML file contents.
+* With the `_path` suffix, Grafana assumes a file path and attempts to read the file from the file system.
+* With the `_url` suffix, Grafana assumes a URL and attempts to load the metadata from the given location.
 
-### Max issue delay
+### Maximum issue delay
 
-Prevention of SAML response replay attacks and internal clock skews between the SP (Grafana), and the IdP is covered. You can set a maximum amount of time between the IdP issuing a response and the SP (Grafana) processing it.
+Prevents SAML response replay attacks and internal clock skews between the SP (Grafana) and the IdP. You can set a maximum amount of time between the IdP issuing a response and the SP (Grafana) processing it.
 
-The configuration options is specified as a duration e.g. `max_issue_delay = 90s` or `max_issue_delay = 1h`
```

Comment:
I like having the extended example here (it saves me a table lookup to figure out the name of the setting), why are we reducing it to only the value?

Reply:
It seemed redundant and unnecessary to me. Since you find it useful, I will add it back in.